src/main/java/seedu/duke/main/CreateName.java

```diff
@@ -1,23 +1,27 @@
 package seedu.duke.main;
 
-import seedu.duke.data.profile.Profile;
 import seedu.duke.data.profile.attributes.Name;
 import seedu.duke.data.profile.utilities.ProfileUtils;
+import seedu.duke.logic.commands.CommandResult;
 import seedu.duke.logic.parser.exceptions.ParamMissingException;
 import seedu.duke.ui.Ui;
 
 /**
  * Creates a Name Class and prompts user for valid name input.
  */
-public class CreateName {
+public class CreateName extends AttributeCreator {
 
-    private Ui ui;
+    public static final String MESSAGE_BYE_DETECTED = "The command word 'bye' is detected.\n "
```

Comment:
Try not to use `\n` and replace with `LS` which is a constant we defined manually, so that the code is standardized :)